### PR TITLE
test(appeals): e2e test for changing the start date of an appeal

### PR DESCRIPTION
**Pull request by RHARV018**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by fabfab-kc — DISMISSED** [review]
LGTM!

File: appeals/e2e/cypress/e2e/back-office-appeals/changeStartDate.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { users } from '../../fixtures/users';
+import { CaseDetailsPage } from '../../page_objects/caseDetailsPage.js';
+import { happyPathHelper } from '../../support/happyPathHelper';
+import { ListCasesPage } from '../../page_objects/listCasesPage';
+import { DateTimeSection } from '../../page_objects/dateTimeSection';
+
+const listCasesPage = new ListCasesPage();
+const dateTimeSection = new DateTimeSection();
+const caseDetailsPage = new CaseDetailsPage();
+//const date = new Date(); // Note: Months are 0-indexed, so 10 is November
```
Comment (PeterBridgerKC) [inline appeals/e2e/cypress/e2e/back-office-appeals/changeStartDate.spec.js:13]: Does this need to be here? Appears to be left over comment from working on `verifyChangeStartDate` method

File: appeals/e2e/cypress/page_objects/caseDetailsPage.js [hunk 2]
```diff
@@ -187,4 +193,18 @@ export class CaseDetailsPage extends Page {
 				);
 		});
 	}
+
+	verifyChangeStartDate() {
+		const date = new Date(); // Note: Months are 0-indexed, so 10 is November
```
Comment (PeterBridgerKC) [inline appeals/e2e/cypress/page_objects/caseDetailsPage.js:198]: I would suggest adding a bit more context here and call the variable something like `dateToday`, as that way it's obvious when it's referenced what the date is.

Where as `date` could be any date, so doesn't give any context when viewed

Comment (RHARV018) [reply appeals/e2e/cypress/page_objects/caseDetailsPage.js]: Thank you, I've just updated and added a new commit

File: appeals/e2e/cypress/e2e/back-office-appeals/changeStartDate.spec.js (hunk 1 above)
Comment (RHARV018) [reply appeals/e2e/cypress/e2e/back-office-appeals/changeStartDate.spec.js:13]: I can't see this on my local code, maybe I deleted it but didn't commit? It's not appearing in my change list though?

**Review by PeterBridgerKC — APPROVED** [review]

**Pull request merged**